packages/adapter/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+import BN from 'bn.js';
+
+export const WEIGHT_REF_TIME_PER_SECOND = new BN(1000000000000);
+export const WEIGHT_PROOF_SIZE_PER_MB = new BN(1024 * 1024);
```

Comment:
Although the values are the same across chains, these are actually chain specific configs.

I think we should define them in chainAdapter such as `this.refTimePerSecond` and `this.proofSizePerMb` so they can be overwritten by each chain if needed.

Reply:
These two values are fixed.

We need to configure `instructionWeight` in the config.